doc-pages/src/static/docs/whats-new.md

```diff
@@ -1,350 +0,0 @@
-# deck.gl v4.0
```

Comment:
Same here, we have a lot of new features to present to our users. It would also follow the pattern from our other frameworks.